### ISSUE TITLE
Question: Any plans for native menu items?

### ISSUE THREAD

**Issue by wcastand** [nodegui/react-nodegui#21]
Hello, look like a really promising project. 

Do you know if it's possible and if it would be hard to provide a way to create native menu like `File` `Edit`, etc ?

Something like this : https://electronjs.org/docs/api/menu

maybe even something like dock menu : 
https://electronjs.org/docs/tutorial/macos-dock

I didn't use Qt in many many years but would making the bindings be a hard thing to do?

**Comment by master-atul** [comment]
Hi @wcastand 
Its not implemented yet, but
Yes, we can support this since Qt supports it.  It will be implemented soon.

**Comment by wcastand** [comment]
is there anyway to help? I won't pretend my 10 years old Qt coding skill will be useful to do it myself but let me know if I can with testing or something else :)

**Comment by master-atul** [comment]
Yes, yes, You can follow the contributors guide to help with the project, docs, testing.
https://nodegui.github.io/nodegui/#/development/README
@wcastand 
It is fairly straightforward to get started. 
I would recommend you start by adding an unexported method to an existing widget.
For example:
You could add the corresponding Qt method to QProgressbar 
https://doc.qt.io/qt-5/qprogressbar.html#textVisible-prop to get a grip on it.

This PR can be used as a guide 
https://github.com/nodegui/nodegui/issues/36
https://github.com/nodegui/nodegui/pull/39

**Comment by master-atul** [comment]
You can always use this issue as a discussion channel if you are in need of help.

**Comment by wcastand** [comment]
I'll probably take a look this weekend then. Thanks for the guide :)
